### PR TITLE
Add `backTitle` property to `BannerDismissalViewController`

### PR DESCRIPTION
**Pull request by ShanMa1991**
### Description
This Pr is to add `backTitle` property to `BannerDismissalViewController` to set the title for back button.

### Implementation
This Pr is to add `backTitle` property to `BannerDismissalViewController` to set the title for back button. When `nil` value is provided, it will show no title. When no value set, it will use the default `Back` title in localized string.

### Screenshots or Gifs
Before:
Before
After, by updating the `BannerDismissalViewController.backTitle`
After

### CODE REVIEW

File: Sources/MapboxNavigation/BannerDismissalViewController.swift [hunk 1]
```diff
@@ -9,6 +9,16 @@ public class BannerDismissalViewController: UIViewController, Banner {
     
     var backButton: BackButton!
     
+    // :nodoc:
+    public var backTitle: String? {
```
Comment (MaximAlien) [inline Sources/MapboxNavigation/BannerDismissalViewController.swift:13]: Thanks! Can you please also add unit-test for that? And for default case as well (when default title is used)?

**Comment by MaximAlien** [comment]
When looking at a screenshot I can see that button might be too small to contain long titles. Would be it be possible to also resize back button width up to some point? I believe we currently use constant for width for that.

File: Sources/MapboxNavigation/BannerDismissalViewController.swift (hunk 1 above)
Comment (ShanMa1991) [reply Sources/MapboxNavigation/BannerDismissalViewController.swift:13]: Thanks for the suggestion, the unit test is added.

File: Sources/MapboxNavigation/BannerDismissalViewController.swift [hunk 2]
```diff
@@ -72,12 +82,17 @@ public class BannerDismissalViewController: UIViewController, Banner {
                                                 comment: "Title of the back button.")
         
         backButton.setTitle(backButtonTitle, for: .normal)
+        backButton.contentEdgeInsets = UIEdgeInsets(top: 0,
```
Comment (MaximAlien) [inline Sources/MapboxNavigation/BannerDismissalViewController.swift:85]: 👍

**Review by MaximAlien — APPROVED** [review]

**Pull request merged**